src/main/java/se/yolean/kafka/keyvalue/ConsumerAtLeastOnce.java

```diff
@@ -270,9 +282,19 @@ void toStats(UpdateRecord update) {
       currentOffsets.get(key).set(update.getOffset());
     }
 
-    var tags = Tags.of("topic", update.getTopic(), "partition", "" + update.getPartition());
```

Comment:
I guess that this is a sign that we should lint/validate against the use of `var` in our java projects?